spyder/config/manager.py

```diff
@@ -15,9 +15,13 @@
 from typing import Any, Dict, Optional, Set
 import weakref
 
+# Third-party imports
+from qtpy.QtWidgets import QMessageBox

```

Comment:
Please move this import to be inside the `Exception` you added below because this file is imported in `spyder/app/start.py` and the idea is to avoid indirect Qt imports there, so Spyder starts quickly.